### ISSUE TITLE
Use the same branch instead of opening multiple PRs

### ISSUE THREAD

**Issue by ChipWolf** [derberg/global-workflows-support#24]
#### Reason/Context

Currently, every time the workflow runs, it opens a new PR on all of our repos.
The behaviour should be similar to most other tools where the same branch is reused/force-pushed to.

**Comment by derberg** [comment]
Hey, it should rather be optional. For us for example it is ok as we remove branch after merging a PR

**Comment by ChipWolf** [comment]
Indeed, but I don't believe any user needs multiple PRs; so, a static branch name should be sufficient, no? @derberg

**Comment by ReenigneArcher** [comment]
I would also like this feature. I use a strategy matrix for this action, and even with using an automerge action it would be nice to have a cleaner history. Adding [concurrency](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#concurrency) to the automerge action seems like a decent approach.

This is a similar action that creates a single PR. https://github.com/BetaHuhn/repo-file-sync-action

The disadvantage of that one being that you have to manually specify each repo you want to sync to and it requires a separate config file.

**Comment by derberg** [comment]
regarding out of date branch, you need to add another workflow that always autoupdates branches. This is how we do it at AsyncAPI -> https://github.com/asyncapi/.github/blob/master/.github/workflows/autoupdate.yml

**Comment by derberg** [comment]
fyi I start working on that

will take the same approach as `sticky comment` action -> https://github.com/marocchino/sticky-pull-request-comment/blob/main/src/comment.ts

dummy but works and battle proved

when the PR is created by this action, it will have a hidden identifier like `<!-- Manage Files In Multiple Repos Action ${id} -->`

so action before creating a new branch, will just check if there are other PRs from the action and checkout the existing branch and push to it

`${id}` is for users that want to make sure that some changes in workflows do not go in a single PR and are still kept separately

---
This new approach will be default and new major will be released

🤞🏼 all works as planned

**Comment by ReenigneArcher** [comment]
@derberg You could just add an input for the branch name, then you don't need to parse the PR bodies and it will save api requests.

**Comment by derberg** [comment]
@ReenigneArcher but the branch name is not predictable, it is generated to make sure it is unique. You mean you want to have an option to override this default branch name generation?

**Comment by derberg** [comment]
@ReenigneArcher no worries. I liked your idea better. Simple and during implementation I could not spot any edge cases that would cause some issues with the action.

PR ready https://github.com/derberg/manage-files-in-multiple-repositories/pull/50
Will release soon and use in my project. In test org all was good. Now time for testing in production 😆
